### DISCUSSION TITLE
Issue with Microsoft Edge

### DISCUSSION THREAD

**Discussion by benley42** [bozdoz/wp-plugin-leaflet-map#130] [Q&A] [answered]
Hi,

I have this problem with Edge (not IE), even if I think no one should use these horrible browser : DOM7009 Unable to decode image at URL.
Is there a way to bypass this issue ?

Thanks in advance,

Ben

**Comment by bozdoz** [comment]
Is this just a basic map shortcode?  [leaflet-map] ?

**Reply by benley42** [reply]
Resolved with adding  "!tap" on my map ! thanks

**Comment by bozdoz** [comment, marked as answer]
Guess this is resolved?
